notebooks/00_preprocess_data.ipynb

```diff
@@ -0,0 +1,494 @@
+    "We will be subsetting the tree data using the following filters:\n",
+    "\n",
+    "- `TREE.STATUSCD` is 1 (live trees)\n",
+    "- `TREE.INVYR` is between 2011 and 2020\n",
+    "- `COND.COND_STATUS_CD` is 1 (forested)\n",
+    "- `COND.CONDPROP_UNADJ` is 1.0 (single-condition plots)\n",
+    "\n",
+    "Assuming that your `SQLite_FIADB_OR.db` file is in the same directory as this notebook, you can execute the following code to extract the tree data from the `TREE` table in the SQLite database:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import modules and set up the connection to the database\n",
+    "import sqlite3\n",
+    "\n",
+    "import pandas as pd\n",
+    "from pyproj import Transformer\n",
+    "\n",
+    "# Connect to the SQLite database\n",
+    "connection = sqlite3.connect(\"./SQLite_FIADB_OR.db\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql = \"\"\"\n",
+    "    -- Get all live tree records for full-plot forested conditions\n",
+    "    -- between 2011 and 2020\n",
+    "    select\n",
+    "      t.CN AS TREE_CN,\n",
+    "      t.PLT_CN,\n",
+    "      cast(t.SPCD as INT) as SPCD,\n",
+    "      t.DIA,\n",
+    "      t.TPA_UNADJ\n",
+    "    from\n",
+    "      tree t\n",
+    "    join\n",
+    "      plot p on t.PLT_CN = p.CN\n",
+    "    join\n",
+    "      cond c on t.PLT_CN = c.PLT_CN and t.CONDID = c.CONDID \n",
+    "    where\n",
+    "      t.STATUSCD = 1\n",
+    "      and c.CONDPROP_UNADJ = 1.0\n",
+    "      and c.COND_STATUS_CD = 1\n",
+    "      and p.INVYR >= 2011\n",
+    "      and p.INVYR <= 2020\n",
+    "\"\"\"\n",
+    "tree_df = pd.read_sql_query(sql, connection)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This should produce a pandas `DataFrame` with n=166,147 records and 5 columns."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(tree_df.head())\n",
+    "print(f\"Tree table shape: {tree_df.shape}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Step 3: Extract the associated plot data and coordinates from the `PLOT` table in the SQLite database"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "FIADB stores fuzzed and swapped coordinates in the `PLOT` table.  As such, this data should not be used in any actual analysis as the coordinates are not accurate. Nonetheless, this workflow will illustrate the needed format of the files.  Note that we are extracting the measurement year (`MEASYEAR`) rather than the inventory year (`INVYR`) as this will be a closer temporal match against spatial data that we use in our modeling workflow.  Run the following cell to create a plot CSV file. "
```

Comment:
I'm not sure how broad of an audience you want to target with this, but if you want to aim for folks with no knowledge of FIA, it might be helpful to add a few details like 1) what are the TREE and PLOT tables and how are they related, and 2) why are coordinates fuzzed?

I don't think you need to give a ton of info or turn this into an FIA 101 tutorial, but just some brief context might make it easier to follow, e.g. that we're loading tree-level inventory data that we'll join to plot locations which are randomly offset in the public dataset to protect confidentiality.

Reply:
This was a good suggestion as well.  I opted to include a brief overview section the details the `PLOT` and `TREE` tables.  It's still a bit light on details, but should address the two comments you raised.